src/components/Strategies/StrategyList.tsx

```diff
@@ -2,75 +2,47 @@ import { Card, Row, Col, Typography } from 'antd';
 import { PlusOutlined } from '@ant-design/icons';
 import styled from 'styled-components';
 
+import { IStrategies } from './../../types';
+import { strategies } from './strategyData';
+
 const { Meta } = Card;
 
 const { Text } = Typography;
 
 function StrategyList() {
+  const strategyClick = (strategy: IStrategies) => {
+    try {
+      (window as any).heap.track('StrategyClicked', {
+        strategy: strategy.title,
+        url: strategy.detailPageURL,
+        id: strategy.itemid,
+      });
+    } catch (e) {
+      console.error(e as any);
+    }
+    const comingSoonTextId = document.getElementById(String(strategy.itemid));
```

Comment:
Why `Id`? Id implies a number and this a HTML element. Just do `comingSoonText` or even `comingSoon`.
You really need to think about proper naming

Reply:
i think i did this because of misunderstanding this https://github.com/chronologic/automate-ui/pull/44#discussion_r864544882 changed it back to strategy.title since unique for each card.